integreat_cms/static/src/js/media-management/component/breadcrumbs.tsx

```diff
@@ -1,3 +1,5 @@
+/* eslint-disable */
+
```

Comment:
Sorry, I don't know how to solve "Do not nest ternary expressions" Error on the line 54 and disabled eslint temporary for this file...

Reply:
I solved this by introducing another variable `const showFullPath = !(searchQuery || mediaFilter);`